pkg/subctl/cmd/diagnose/firewall_vxlan.go

```diff
@@ -44,16 +44,6 @@ func init() {
 	addDiagnoseFWConfigFlags(command)
 	addVerboseFlag(command)
 	diagnoseFirewallConfigCmd.AddCommand(command)
-
```

Comment:
Should update one more place - e2e test to use new cmd:
https://github.com/submariner-io/submariner-operator/blob/e40739bdda7a3b70f1d29b79f672689c2dbe781c/scripts/kind-e2e/e2e.sh#L60